fritz

```diff
@@ -72,6 +62,9 @@ def generate_skyportal_token():
 
 
 def check_config(cfg='fritz.defaults.yaml', yes=False):
+    """
```

Comment:
Do these config files get left behind outside the container after a user stops a container? Might be good to have them generated in a temporary directory and the copied into the container so that the default config file always remains unchanged

Reply:
The default config stays untouched. When you do `fritz run --init`, if you don't have `fritz.yaml`, `fritz` will ask if you want to `cp fritz.defaults.yaml fritz.yaml` warning that it is not recommended. `fritz.yaml` then gets dissected and distributed to K and SP ([here](https://github.com/fritz-marshal/fritz/blob/76b8429a277a555e05a3b9917caa822191ab8254/fritz#L91)).